### PR TITLE
chore: add China mirroring service section

### PR DESCRIPTION
**Pull request by septs**
CC @ZenithalHourlyRate

### CODE REVIEW

**Comment by septs** [comment]
CC @votdev

File: installation/on_debian.rst [hunk 1]
```diff
@@ -18,6 +18,9 @@ below.
 On `Raspberry Pi OS <https://www.raspberrypi.org/software/operating-systems/>`_ the below
 instructions only partially work. Please refer to a specific `installation script <https://github.com/OpenMediaVault-Plugin-Developers/installScript>`_
 
+If you are a user in mainland China, TUNA provides mirroring services.
```
Comment (votdev) [inline installation/on_debian.rst:21]:
```suggestion
If you are a user in mainland China, TUNA provides `mirroring services <https://mirrors.tuna.tsinghua.edu.cn/help/openmediavault/>`_.
```

File: installation/on_debian.rst [hunk 2]
```diff
@@ -18,6 +18,9 @@ below.
...
+Please see <https://mirrors.tuna.tsinghua.edu.cn/help/openmediavault/>
```
Comment (votdev) [inline installation/on_debian.rst:22]: Remove this line.

**Review by votdev — CHANGES_REQUESTED** [review]

File: installation/on_debian.rst [hunk 3]
```diff
@@ -18,6 +18,8 @@ below.
 On `Raspberry Pi OS <https://www.raspberrypi.org/software/operating-systems/>`_ the below
 instructions only partially work. Please refer to a specific `installation script <https://github.com/OpenMediaVault-Plugin-Developers/installScript>`_
 
+If you are a user in mainland China, TUNA provides `mirroring services <https://mirrors.tuna.tsinghua.edu.cn/help/openmediavault/>`.
```
Comment (votdev) [inline installation/on_debian.rst:21]: The `_` is missing which will result in a render error.
```suggestion
If you are a user in mainland China, TUNA provides `mirroring services <https://mirrors.tuna.tsinghua.edu.cn/help/openmediavault/>`_.
```

**Review by votdev — CHANGES_REQUESTED** [review]

**Comment by votdev** [comment]
Thanks for the origin PR. Superseded by 1b9fe3eb2f4bdca327dc3969d505e7ef6c02d6c5.

**Pull request closed without merging**